script/upload.py

```diff
@@ -72,6 +72,8 @@ def main():
     # Press the publish button.
     publish_release(github, release['id'])
 
+    # TODO: run publish-to-npm script here
```

Comment:
The plan is to keep this as-is and run the script manually for now. The comment here is just a note for when we're ready to chain it into the publishing process in the future.